s3/replication/common/tests/system/test_head_object.py

```diff
@@ -50,10 +50,16 @@ async def main():
     bucket_name = config.source_bucket_name
     request_id = "dummy-request-id"
 
```

Comment:
Add line here

```
# For Cortx, Default value of version id=None
# For AWS, please provide the version id
version_id = None
```

Remove comments from below and pass version_id while using the class, rather than passing None.

Reply:
done